source/transfer_list.rst

```diff
@@ -994,6 +995,177 @@ into its memory map during platform setup. If other memory types are required
      - hdr_size + 0x8
      - The size of the memory region.
 
+.. _opendice_entries:
+
+Entries related to Open DICE
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+These entries contain [Open-DICE]_ information elements that must be transferred between boot stages in a DICE architecture where a DICE Protection Environment (DPE) is not used.
+
+This includes the Compound Device Identifier (CDI) used for attestation and the CDI used for sealing, as well as the accumulated certificate chain.
+
+.. note::
+  As described in [DICE]_, CDIs are very sensitive data and are as long-lived as the layers they represent.
+  Each layer is therefore responsible for the proper handling of the CDIs it owns, as well as the derived cryptographic key material.
+  In particular, any given layer is expected to use and erase the CDI it received.
+  While the CDI is used, the layer needs the ability to prevent access to the CDI and disclosure of the value.
+
+The following entry types are currently defined:
+
+**CDI for attestation (XFERLIST_OPENDICE_CDI_ATTEST)**
+
+This entry type holds the "Attestation CDI" described in [Open-DICE]_.
+
+The CDI computed by the Sender's boot stage SHALL be added to the TL.
+The CDI received from the previous boot stage (i.e., DICE layer) MUST NOT be forwarded.
+
+.. _tab_dice_cdi_attest:
+.. list-table:: DICE attestation CDI
+   :widths: 2 2 2 8
+
+   * - Field
+     - Size (bytes)
+     - Offset (bytes)
+     - Description
+
+   * - tag_id
+     - 0x3
+     - 0x0
+     - The tag_id field must be set to **0x200**.
+
+   * - hdr_size
+     - 0x1
+     - 0x3
+     - |hdr_size_desc|
+
+   * - data_size
+     - 0x4
+     - 0x4
+     - The size (in bytes) of the CDI
+
+   * - profile_id
+     - 0x4
+     - 0x8
+     - the Open DICE profile identifier (See :numref:`opendice_profile_id` for the allowed values)
+
+   * - cdi_attest
+     - data_size - 0x4
```

Comment:
Are CDIs larger than 4 bytes?
If so the 0x4 is a typo?

The same question holds for lines 1093 and 1039.

Reply:
> Are CDIs larger than 4 bytes? If so the 0x4 is a typo?
> 
> The same question holds for lines 1093 and 1039.

I am probably missing something fundamental about the notation conventions.  My reasoning was: the data (`data_size` long) comprises `profile_id` (fixed length, 4 bytes) and `cdi_attest` (variable length).  Therefore, `cdi_attest`'s length is `data_size - 0x4`.

Reply:
Hi Thomas,

I think I was the one getting confused with the "-" and assumed the line to state something like "data_size, this should be  0x4".
All checks out. I'll resolve this converstation.